### DISCUSSION TITLE
Multiple Inverters

### DISCUSSION THREAD

**Discussion by neilrfuller** [springfall2008/batpred#1185] [Q&A]
Hi, I have two x Solis S5-EH1P3.6K-L each with 4kW panels. Waveshare devices and wills106/homeassistant-solax-modbus working fine. Could you advise me if it's possible to configure predbat for a twin inverter set up please?

**Comment by gcoan** [comment]
I have twin GivEnergy inverters and it works fine for me.  You just duplicate the inverter control lines for charging, discharging, mode, etc in apps.yaml
This assumes you have two sets of controls in Home Assistant for the two inverters.  There may be limitations/configuration of your Solis needed to achieve this, I can’t help on that specific sorry
